service/src/main/kotlin/app/cash/backfila/service/runner/statemachine/BatchPrecomputer.kt

```diff
@@ -59,6 +67,13 @@ class BatchPrecomputer(
           backfillRunner.factory.transacter.transaction { session ->
             val dbRunPartition = session.load(backfillRunner.partitionId)
             dbRunPartition.precomputing_done = true
+
+            session.save(DbEventLog(
+                backfillRunner.backfillRunId,
+                partition_id = backfillRunner.partitionId,
+                type = DbEventLog.Type.STATE_CHANGE,
+                message = "precomputing complete"
+            ))
           }
           logger.info { "Precomputing completed for ${backfillRunner.logLabel()}" }
```

Comment:
Add elapsed here too?

Reply:
elapsed is just for the current batch so it wouldn't be relevant when you completed all precomputing